src/controllers/timeZoneAPIController.js

```diff
@@ -6,13 +6,12 @@ const timeZoneAPIController = function () {
     const premiumKey = process.env.TIMEZONE_PREMIUM_KEY;
     const commonKey = process.env.TIMEZONE_COMMON_KEY;
     if (!req.body.requestor.role) {
-      res.status(403).send('Unauthorized Request');
```

Comment:
Could you explain what are those changes for?

Reply:
It is not related to my PR, I  don't know how this came to here